pcdsdevices/tests/test_attenuator.py

```diff
@@ -216,3 +216,50 @@ def test_new_attenuator_smoke(fake_new_attenuator):
             fake_new_attenuator.status_info()
         )
     )
+
+
+@pytest.fixture(scope="function")
+def at2l0(request):
+    at2l0 = make_fake_device(AT2L0)("AT2L0:", name=f"fake_at2l0_for_{request.function}")
+    for walk in at2l0.walk_signals():
+        try:
+            if "message" in walk.dotted_name:
+                walk.item.sim_put("")
+            else:
+                walk.item.sim_put(0)
+        except AttributeError:
+            ...
+        walk.item._metadata["severity"] = 0
+    return at2l0
+
+
+def test_at2l0_error_summary(at2l0):
+    assert at2l0.error_summary.get() == "No Errors"
+    at2l0.blade_05.state.error_message.sim_put("test error")
+    assert "test error" in at2l0.error_summary.get()
+
+
+def test_at2l0_error_bitmask(at2l0):
+    assert at2l0.error_summary_bitmask.get() == 0
+    # blade_01 intentionally ignored (mirror)
+    at2l0.blade_01.motor.plc.err_code.sim_put(1)
+    assert at2l0.error_summary_bitmask.get() == 0
+    at2l0.blade_19.motor.plc.err_code.sim_put(1)
+    # bitmask intentionally reversed (better for ui)
+    assert at2l0.error_summary_bitmask.get() == 0b1
+    at2l0.blade_17.motor.plc.err_code.sim_put(1)
+    assert at2l0.error_summary_bitmask.get() == 0b101
+
+
+def test_at2l0_clear_errors(at2l0):
+    samples = (
+        at2l0.blade_01.motor.plc.cmd_err_reset,
```

Comment:
There's one of these for every blade right?  Is there a reason we don't just... test them all?

Reply:
mostly because I didn't want to write a reflexive test (checking all the pvs in at2l0's list of pvs to check, for example), and I didn't feel like copy/pasting 20 times

maybe I should suck it up and check them all

Reply:
I was just thinking something like 
```python
for i in range(20):
    samples.append(getattr(at2l0, f'blade_{str(i).zfill(2)}.motor.plc.cmd_err_reset'))
```

Or is there something that I'm missing there

Reply:
yeah I can do this